app/javascript/packs/tasklist.js

```diff
@@ -0,0 +1,3 @@
+window.addHighlight = function(elementId) {
+  document.getElementById(elementId).classList.add('tasklist_item_highligh');
```

Comment:
I think for consistency it'll likely be neater and more manageable if we use Stimulus JS throughout to organise JS. Would basically just require adding a `tasklist_controller.js` with a `connect` or `initialize` function that does this and then attaching it as the `data-controller` html attribute on your <a> tag.

Reply:
Yup, can do. I'll wait for you to merge your PR first and will fix this